src/backend/TrafficCourts/Messaging/MessageContracts/DispuantEmailFiltered.cs

```diff
@@ -0,0 +1,30 @@
+﻿using TrafficCourts.Common.Features.Mail;
+
+namespace TrafficCourts.Messaging.MessageContracts
+{
+    /// <summary>
+    /// Raised when an email was filtered before being sent due to allow list processing.
+    /// </summary>
+    public class DispuantEmailFiltered
+    {
+        /// <summary>
+        /// The email message.
+        /// </summary>
+        public EmailMessage Message { get; set; }
```

Comment:
Getting a non-null warning here.  Should there be a constructor to initialize this property?